server/blob/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
usually, we don't create this intermediate level of pom (I don't know why). Maybe we should stick to what was done until now ?

Reply:
We are doing this when introducing new branches in the project tree.
Here are two examples:
- backend-common
- server/protocols/jmap-integration-testing/

Reply:
ok but it's not the case in server/pom.xml it's why I told you

Reply:
The difference is that server/pom.xml was not introduced recently.